pom.xml

```diff
@@ -119,7 +119,7 @@
     <sparkbundle.version></sparkbundle.version>
     <flink.version>1.14.3</flink.version>
     <spark2.version>2.4.4</spark2.version>
-    <spark3.version>3.2.0</spark3.version>
+    <spark3.version>3.2.1</spark3.version>
```

Comment:
there is also parquet and orc version upgrades
https://spark.apache.org/releases/spark-release-3-2-1.html
@YannByron shall we keep these aligned too ?

Reply:
yep. i will.